tests/integration/test_sns.py

```diff
@@ -1533,41 +1532,41 @@ def test_publish_batch_exceptions(
     def test_publish_sqs_from_sns_with_xray_propagation(
         self, sns_client, sns_create_topic, sqs_client, sqs_create_queue, sns_subscription
     ):
-        # TODO: remove or adapt for asf
-        if SQS_BACKEND_IMPL != "elasticmq":
-            pytest.skip("not using elasticmq as SQS backend")
-
         def add_xray_header(request, **kwargs):
             request.headers[
                 "X-Amzn-Trace-Id"
             ] = "Root=1-3152b799-8954dae64eda91bc9a23a7e8;Parent=7fa8c0f79203be72;Sampled=1"
 
-        sns_client.meta.events.register("before-send.sns.Publish", add_xray_header)
+        try:
+            # TODO this needs to be removed again!
```

Comment:
what does this depend on? (when can we remove it?)

Reply:
Ah, this is a leftover from analyzing / fixing the test :) Thanks!